src/pages/index.jsx

```diff
@@ -21,8 +22,11 @@ export default function Index({
   values,
   configs: { navbarConfig, faqConfig },
 }) {
+  const [loaded, setLoaded] = useState(false)
+  useEffect(() => setTimeout(() => setLoaded(true), 3500), [])
```

Comment:
Is 3.5 seconds a bit long? I think around 0.5 seconds would be enough. What do you think?

Reply:
Yeah about that, seems like what @rmehri01 mentioned above, it doesn't start loading the main components until `loaded` evaluates to `true`. Still investigating that but I couldn't find any example online that could be useful in this case :( Seems like you were able to optimize loading time so we can hold onto this for now! Thanks a lot 🙏